.travis.yml

```diff
@@ -0,0 +1,56 @@
+sudo: false
```

Comment:
Interesting. Travis usually detects that the script uses `sudo` and does this automatically.